templates/testpmd/resource-specs/sriov-vm1-template.yaml

```
@@ -0,0 +1,109 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: dpdk-vm-template
+objects:
+- apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    annotations:
+      openshift.io/display-name: "RHEL VM with SR-IOV-backed NIC and DPDK."
+      description: >-
+        Template for RHEL 8.4 VM, with a secondary interface backed by an SR-IOV node NIC.
+        Involves replacing the default NIC driver (iavf) to vfio-pci, to enable running DPDK.
+      openshift.io/documentation-url: "https://github.com/RHsyseng/cnv-supplemental-templates/tree/main/templates/testpmd"
+    name: ${VM_NAME}
+    namespace: ${WORKING_NAMESPACE}
+  spec:
+    dataVolumeTemplates:
+    - apiVersion: cdi.kubevirt.io/v1beta1
+      kind: DataVolume
+      metadata:
+        name: ${DV_NAME}
+      spec:
+        source:
+          registry:
+            pullMethod: node
+            url: ${IMAGE_URL}
+        storage:
+          resources:
+            requests:
+              storage: ${REQUESTED_DV_STORAGE_SIZE}
```

```suggestion
              storage: ${DV_SIZE}
              
```
param names should be kept simple

---

I agree - simple and clarify what the parameter is used for.

---

simple and short :)